docs/src/routes/docs/components/dialog.svx

```diff
@@ -0,0 +1,101 @@
+---
+name: Dialog
+shadowRaised: '0 2px 4px -1px transparentize(black, .8), 0 3px 5px 0 transparentize(black, .86), 0 1px 10px 0 transparentize(black, .88);'
+---
+
+<script>
+  import { Dialog, Button } from 'attractions';
+  import { AlertTriangleIcon } from 'svelte-feather-icons';
+  import Showcase from 'src/containers/docs/showcase.svelte';
+  import ColorPreview from 'src/components/docs/color-preview.svelte';
+  import ShadowPreview from 'src/components/docs/shadow-preview.svelte';
+</script>
+
+# Dialog
+
+A modal window that informs users about a task and can contain critical information or require actions.
+
+<Showcase>
+  <div slot="showcase" class="padded">
+    <Dialog title="Are you sure you want to exit?" danger closeCallback={() => console.log('cancelled')}>
+      <div slot="title-icon">
+        <AlertTriangleIcon size="25" />
+      </div>
+      <p>You have not saved your changes yet.</p>
+      <p>If you exit without saving, changes will be lost.</p>
+      <p>Are you sure you want to exit?</p>
+      <Button>Yes, exit</Button>
```

Comment:
The implicit style guidelines suggest that we write the button texts all lowercase, so let's stick to that in the docs